### PR TITLE
image orientation change-step one

### PR DESCRIPTION
**Pull request by ancafarcas**

### CODE REVIEW

File: superdesk/media/image.py [hunk 1]
```diff
@@ -44,8 +56,13 @@ def get_meta(file_stream):
                 value = {ExifTags.GPSTAGS[vk].strip(): vv for vk, vv in v.items()}
                 exif_meta[key] = value
             else:
-                value = v.decode('UTF-8') if isinstance(v, bytes) else v
-                exif_meta[key] = value
```
Comment (nistormihai) [inline superdesk/media/image.py:48]: I don't think is a good placement here, it should be in a different method and also not a `getter` method.

Comment (nistormihai) [inline superdesk/media/image.py:48]: please commit to the `superdesk/superdesk-core` fist

Comment (ancafarcas) [inline superdesk/media/image.py:48]: I don't have rights to commit on `superdesk/superdesk-core`

Comment (ancafarcas) [inline superdesk/media/image.py:48]: @nistormihai you're right about the placement of the method. I changed its place. 
The new PR is available on the `superdesk/superdesk-core` repo:
https://github.com/superdesk/superdesk-core/pull/411

**Pull request closed without merging**